src/routes/auth.py

```diff
@@ -76,6 +76,8 @@ async def refresh_token(credentials: HTTPAuthorizationCredentials = Security(oau
         token = credentials.credentials
     else:
         token = credentials
+async def refresh_token(credentials: HTTPAuthorizationCredentials = Security(oauth2_scheme), db: Session = Depends(get_db)):
+    token = credentials.credentials
```

Comment:
it is obsolete code - we modified it a couple pull ago. I will include it but please refresh your branches